### ISSUE TITLE
Content Review for Outreach & Events MVP from Public Websites Team

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#34253]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [ ] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [X] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [X] Content support
  - [ ] IA support
  
*Please describe your need and provide any additional documents and/or links that will help us help you. If we need to add a react widget to a page, be sure to provide the code here.*

  
## Supporting Artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:[Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/outreach-events/product-outline.md)
- Link to or attach user flows: [Product Design Library](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/teams/public-websites/outreach-and-events/design)
- Link to prototype or mockups: [Sketch file](https://sketch.com/s/8ec18b80-a905-47fb-a217-5720450befa1)
- Link to any epics/issues that may be helpful: 
- Context for this request:  We're looking for some focused verbiage as it pertains to recurring events as this is new to VA.gov and Outreach & Events and want to ensure we're communicating the appropriate actions for users. @callen2563 will post screenshots with specifics.

## Will this work be going through the Collaboration Cycle?
- [X] Yes (Midpoint complete, preparing for Staging Review)
- [ ] No

## When does this work need to be done?
- Estimated launch date: January 17, 2022
- Estimated staging review date: January 13, 2022
- Content and IA work needed by: December 21, 2021 (This will impact FE build requirements) 

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner: David Conlon
-	Product manager: Brian Lloyd
-	Designer: Cassandra Allen
-	FE engineer: Kelson Adams
-	Product/team Slack channel: #vsa-public-websites

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) Slack channel and tag @RLHecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by callen2563** [comment]
@RLHecht Update since you last reviewed copy: We have just finished the first round of research. For MVP we were told to keep most of the copy the same to what is seen in [production](https://www.va.gov/outreach-and-events/events/). For example "see all events" link and register buttons. However, we did change the button copy for the button under reoccurring events from "check more times" to "show more times" since this button reveals more times in the list. I have attached a screen shot. 
![Outreach   Events - Mobile - detail - more occurences](https://user-images.githubusercontent.com/68560244/146421655-82902177-fd43-496f-acc3-c7f90fc7c6ca.png)
![Outreach   Events - Mobile - detail - more occurences - open](https://user-images.githubusercontent.com/68560244/146422205-d927142a-550f-4614-a574-2db7fed7b965.png)
